go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rancher/rancher
 
-go 1.22
+go 1.22.0
```

Comment:
Why is this added? Just `go 1.22` should be here.

Reply:
This is due to this [issue](https://github.com/bazelbuild/rules_go/issues/3850)

Reply:
I will remove the go.mod and go.sum from my review.